packages/ra-ui-materialui/src/button/ExportButton.tsx

```diff
@@ -101,7 +101,7 @@ interface Props {
     label?: string;
     maxResults?: number;
     onClick?: (e: Event) => void;
-    resource?: string;
+    resource: string;
```

Comment:
No, it's not required at compile time, because it's injected at runtime

Reply:
I was using type-script with strictMode = true. I'll change it.